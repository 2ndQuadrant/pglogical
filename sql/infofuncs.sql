DO $$
BEGIN
	IF (SELECT setting::integer/100 FROM pg_settings WHERE name = 'server_version_num') = 904 THEN
		CREATE EXTENSION IF NOT EXISTS pglogical_origin;
	END IF;
END;$$;
CREATE EXTENSION pglogical;

SELECT pglogical.pglogical_max_proto_version();

SELECT pglogical.pglogical_min_proto_version();

SELECT pglogical.pglogical_version() = extversion
FROM pg_extension
WHERE extname = 'pglogical';

DROP EXTENSION pglogical;
