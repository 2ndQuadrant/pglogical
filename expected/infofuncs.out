DO $$
BEGIN
	IF (SELECT setting::integer/100 FROM pg_settings WHERE name = 'server_version_num') = 904 THEN
		CREATE EXTENSION IF NOT EXISTS pglogical_origin;
	END IF;
END;$$;
CREATE EXTENSION pglogical;
SELECT pglogical.pglogical_max_proto_version();
 pglogical_max_proto_version 
-----------------------------
                           1
(1 row)

SELECT pglogical.pglogical_min_proto_version();
 pglogical_min_proto_version 
-----------------------------
                           1
(1 row)

SELECT pglogical.pglogical_version() = extversion
FROM pg_extension
WHERE extname = 'pglogical';
 ?column? 
----------
 t
(1 row)

DROP EXTENSION pglogical;
